src/CGRANode.cpp

```diff
@@ -576,6 +646,8 @@ void CGRANode::disableAllFUs() {
   m_canMAC = false;
   m_canLogic = false;
   m_canBr = false;
+  m_canLut = false;
   m_supportComplex = false;
+  for (int i = 0; i < MAXIMUM_COMBINED_TYPE; i++) m_supportComplexType[i] = false; 
```

Comment:
Plz break this into two lines with `{ }`.

Reply:
I deleted this line.